src/content/en/2019/fonts.md

```diff
@@ -0,0 +1,121 @@
+
+This is perhaps unsurprising given Google Fonts dominance in `06.02`, as 75% Google Fonts requests-dominance would all be different host requests. Note that `preload` is not yet available with Google Fonts.
+
+**06.03:  MIME types**
+
+WOFF2 is pretty well supported in web browsers today. Google Fonts serves WOFF2, a format that offers improved compression over its predecessor. To me this is perhaps supporting evidence for a case to be made to go WOFF2-only for web fonts. I wonder where the double-digit WOFF usage is coming from—perhaps Internet Explorer?
+
+Third place `octet-stream` (and `plain` a little further down) would seem to suggest that a lot of web servers are misconfigured to not properly send a MIME type with web font file requests.
+
+**06.36–38:  src formats included in @font-face blocks**
+
+I was hoping to see [SVG fonts](https://caniuse.com/#feat=svg-fonts) on the decline. They’re buggy and implementations have been removed from every browser except Safari. Time to drop these, y’all.
+
+**06.39–41:  src formats combinations**
+
+- [ ] TODO *Zach’s note: The order here is important, yeah.*
+
+**06.04:  font-display Usage**
+
+I assume this will be creeping up over time, especially now that [Google Fonts is adding `font-display` to all new code snippets](https://www.zachleat.com/web/google-fonts-display/) copied from their site. If you’re using Google Fonts—update your snippets! If you’re not using Google Fonts—use `font-display`! Read more about [`font-display` on MDN](https://developer.mozilla.org/en-US/docs/Web/CSS/@font-face/font-display).
+
+**06.44: font-display values**
+
+As an easy way to show fallback text while a web font is loading, `font-display: swap` reigns supreme and is the most common descriptor value. `swap` is also the default descriptor value used by new Google Fonts code snippets too. I would have expected `optional` (only render if cached) to have a bit more usage here as a few prominent developer evangelists lobbied for it a bit—but no dice.
+
+**06.09a: Web font requests per page**
+
+The median web page makes 4 web font requests. Even the 10th percentile requests one. The ubiquity of Web fonts is complete. It does seem interesting to me that web font requests seem to be pretty steady across desktop and mobile. I’m glad to see the recommendation to hide `@font-face` blocks inside of a `@media` query didn’t catch on (don’t do this).
+
+**06.34: Font requests per page**
+
+Why are there more requests for fonts made on mobile devices? Is it related to fewer typefaces available on mobile devices, which in turn means fewer `local()` hits in Google Fonts CSS? Actually we might be on to something here.
+
+**06.09b: Web font families**
+
+- [ ] TODO *Zach’s note: Is this families per page? What is `httparchive.almanac.parsed_css`?*
```

Comment:
06_09a is the number of font requests per page
06_09b is the number of `@font-face` declarations per page

`parsed_css` is the table containing tokenized stylesheets, so we can more easily/accurately query for specific font declarations/styles/values.

Reply:
You shouldn't have to worry about elaborating on how we arrived at these numbers (analytical method, etc) but if needed you can direct readers to our [Methodology](https://docs.google.com/document/d/1ViWF_7if4ATds36qoTJYUL3gxIigYG1jsylMqZERuaM/edit?usp=sharing) page (will have its own URL at launch) which contains more info about the CSS parsing.

Reply:
Sorry @rviscomi one question here: `"The median desktop page declares 3 distinct font families (@font-face declarations)`. Is this distinct font families or @font-face blocks? A @font-face block can share the same font-family with another block and have distinct style or weight.

Reply:
It represents each literal `@font-face` declaration